sdk/keyvault/keyvault-keys/tests/utils/crypto.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { isNode } from "./recorder";
+
+export function str2ab(str: string): Uint8Array {
```

Comment:
This might be a silly question, but what's an `ab`?

Reply:
I'm thinking maybe it's ArrayBuffer and I'm not quite awake :D

Reply:
I changed these functions to be more explicit. Namely: stringToUint8Array and uint8ArrayToString.